<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>esistenza_all</class>
 <widget class="QDialog" name="esistenza_all">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>567</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>741</width>
    <height>567</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>741</width>
    <height>567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="7">
    <widget class="QwtPlot" name="esistenza">
     <property name="canvasBackground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QCheckBox" name="es_prvev">
        <property name="text">
         <string>Visualizza esistenza preventivato</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Seleziona anno</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mese">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gennaio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Febbraio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Marzo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Aprile</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Maggio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Giugno</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Luglio</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Agosto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Settembre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ottobre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Novembre</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dicembre</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="ft_acq">
        <property name="text">
         <string>Visualizza fatturato acquisto</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="3">
       <widget class="QCheckBox" name="ft_vn">
        <property name="text">
         <string>Visualizza fatturato di vendita</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Seleziona mese</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="anno"/>
      </item>
      <item row="5" column="0" colspan="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2" colspan="3">
       <widget class="QCheckBox" name="es_ord">
        <property name="text">
         <string>Visualizza esistenza ordinato</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Esporta in pdf</string>
        </property>
        <property name="icon">
         <iconset resource="../fabaria.qrc">
          <normaloff>:/images/pdf.png</normaloff>:/images/pdf.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QLabel" name="label_stat">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QPushButton" name="esci">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Esci</string>
        </property>
        <property name="icon">
         <iconset resource="../fabaria.qrc">
          <normaloff>:/images/dialog-close.png</normaloff>:/images/dialog-close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>../qwt/qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../fabaria.qrc"/>
 </resources>
 <connections/>
</ui>
